pkg/v22/controllercontext/status.go

```diff
@@ -0,0 +1,9 @@
+package controllercontext
+
+type ContextStatus struct {
+	Worker ContextStatusWorker
+}
+
+type ContextStatusWorker struct {
+	Nodes int32
```

Comment:
Worker node count will be used in the `clusterconfigmap` resource to set the replica count for ingress controller.